docs/resources/Template.md

```diff
@@ -0,0 +1,114 @@
+# Template Resource
+
+### Template Object
+
+Represents a code that when used, creates a guild based on a snapshot of an existing one.
+
+###### Template Structure
+
+| Field                   | Type                                                             | Description                                            |
+| ----------------------- | ---------------------------------------------------------------- | ------------------------------------------------------ |
+| code                    | string                                                           | the template code (unique ID)                          |
+| name                    | string                                                           | template name                                          |
+| description             | string                                                           | the description for the template                       |
```

Comment:
```suggestion
| description             | ?string                                                          | the description for the template                       |
```

If you provide no description, the description field can be null:

![Postman](https://user-images.githubusercontent.com/46497296/95003558-d5a36300-05e0-11eb-9304-7667d88292ef.png)